cpu/lpc2387/clocks.c

```diff
@@ -43,14 +43,14 @@ static inline void pllfeed(void)
  * @brief   Enabling MAM and setting number of clocks used for Flash memory fetch
  * @internal
  */
-static inline void init_mam(void)
+void cpu_init_mam(void)
 {
     MAMCR  = 0x0000;
     MAMTIM = 0x0003;
     MAMCR  = 0x0002;
 }
 
-static void init_clks1(void)
+void cpu_init_pll(void)
 {
     /* Disconnect PLL */
     PLLCON &= ~0x0002;
```

Comment:
Care to also change `while (PLLSTAT & BIT25);` to `while (PLLSTAT & BIT25) {}` in line 59? And same style fix in line 65, line 70, line 93, and line 100?